CHANGELOG.rst

```diff
@@ -4,6 +4,10 @@ Changelog
 in development
 --------------
 
+* Added test coverage and test timing capabilities to ``st2-run-pack-tests``.
```

Comment:
Nit-picky, but can you re-locate this to the bottom. New entries traditionally go in at the bottom of the change log.